extensions/aerial-photo/examples/item.json

```diff
@@ -19,14 +19,7 @@
   "assets": {
     "visual": {
       "href": "./72360.tiff",
-      "type": "image/tiff; application=geotiff; profile=cloud-optimized",
-      "file:checksum": "1220b7deb18ad9dc6f3f94df60c26dd235a019946b8b6b7d1a36f100a8f9f1889130",
-      "eo:bands": [
-        {
-          "name": "gray",
-          "common_name": "pan"
-        }
-      ]
+      "type": "image/tiff; application=geotiff; profile=cloud-optimized"
```

Comment:
Does this change belong with the rest? Looks like it's just removing some unnecessary content.

Reply:
I was updating all the examples for the collections and I realised the item example was wrong.

Reply:
I'll put these item example changes into a separate PR tomorrow :thumbsup:

Reply:
Done!